### DISCUSSION TITLE
Is it currently possible to target multiple hosts?

### DISCUSSION THREAD

**Discussion by edb34947** [FantasticFiasco/serilog-sinks-http#270] [Q&A] [answered]
I am running a 3-node logstash and 3-node elastic cluster and would like to be able to specify all of the logstash hosts up front that I want my logs shipped to. 

Is there a way to do this currently that would load-balance between the targets?

**Comment by FantasticFiasco** [comment, marked as answer]
Not using this sink. It's been somewhat discussed in #177, and there are better network solutions for this out there that would't involve any implementation in this library.

**Reply by edb34947** [reply]
That's fair, I hadn't spotted #177 but I would tend to agree with the conclusions drawn there.

**Reply by FantasticFiasco** [reply]
No problem! I hope that you find a suitable solution to your problem.
